### PR TITLE
Fix: timeouts of waiting on extensions complete installation RecommendedExtensions test

### LINKED ISSUE #22502: Fix timeouts of waiting on extensions complete installation in 'Java 11 with Quarkus' RecommendedExtensions test
### Is your task related to a problem? Please describe

'Java 11 with Quarkus' RecommendedExtensions test is waiting on extensions complete installation 20 minutes which is too long
https://main-jenkins-csb-crwqe.apps.ocp-c1.prod.psi.redhat.com/job/Testing/job/e2e/job/basic/job/typescript-tests/11671/console
```
20:00:16        • Check if {"publisher":"redhat","name":"java"} are installed.
20:00:16            ▼ extensionSection.findItem(java).
20:20:26            ▼ ActivityBar().getViewControl("Extensions"))?.openView(): open Extensions view.
20:20:26            ▼ extensionsView?.getContent().getSections(): get current section.
20:20:26              ‣ DriverHelper.waitAllPresence - By(css selector, .name)
20:20:26        • Check if {"publisher":"redhat","name":"java"} are installed.
20:20:26            ▼ extensionSection.findItem(java).
20:40:20            ▼ ActivityBar().getViewControl("Extensions"))?.openView(): open Extensions view.
20:40:20            ▼ extensionsView?.getContent().getSections(): get current section.
20:40:20              ‣ DriverHelper.waitAllPresence - By(css selector, .name)
20:40:20        • Check if {"publisher":"redhat","name":"java"} are installed.
20:40:20            ▼ extensionSection.findItem(java).
20:57:37  Cancelling nested steps due to timeout
```

Command to run the test:
> docker run --shm-size=4096m -p 5920:5920 -e TS_SELENIUM_LOAD_PAGE_TIMEOUT=420000 -e TS_SELENIUM_BASE_URL=https://devspaces.apps.ocp413.crw-qe.com/ -e DELETE_WORKSPACE_ON_FAILED_TEST=true -e TS_SELENIUM_START_WORKSPACE_TIMEOUT=120000 -e NODE_TLS_REJECT_UNAUTHORIZED=0 -e VIDEO_RECORDING=true -e E2E_OCP_CLUSTER_API_URL=https://api.ocp413.crw-qe.com:6443/ -e TS_SELENIUM_LOG_LEVEL=TRACE -e TS_WAIT_LOADER_PRESENCE_TIMEOUT=120000 -e TS_COMMON_DASHBOARD_WAIT_TIMEOUT=30000 -e OCP_INFRA=PSI -v /tmp:/usr/local/bin:Z -v /home/hudson/.kube:/home/seluser/.kube:Z -v /mnt/hudson_workspace/workspace/Testing/e2e/basic/typescript-tests/test-repo/tests/e2e:/tmp/e2e:Z -v /mnt/hudson_workspace/workspace/Testing/e2e/basic/typescript-tests/test-repo/tests/e2e/video:/tmp/ffmpeg_report:Z -e USERSTORY=RecommendedExtensions -e TS_SELENIUM_VALUE_OPENSHIFT_OAUTH=true -e TS_OCP_LOGIN_PAGE_PROVIDER_TITLE=htpasswd -e TS_SELENIUM_OCP_USERNAME=admin -e 'TS_SAMPLE_LIST=Java 11 with Quarkus' -e TS_COMMON_DASHBOARD_WAIT_TIMEOUT=120_000 quay.io/eclipse/che-e2e:next

### Describe the solution you'd like

Fix timeouts of waiting on extensions complete installation in 'Java 11 with Quarkus' RecommendedExtensions test.

### Describe alternatives you've considered

_No response_

### Additional context

The is the same problem with 'Java 11 with Lombok' RecommendedExtensions test: https://main-jenkins-csb-crwqe.apps.ocp-c1.prod.psi.redhat.com/job/Testing/job/e2e/job/complex/job/latest/job/functional-tests/893/

### PR DESCRIPTION
**Pull request by nallikaea**
### What does this PR do?

- Fix issue  #22502 
because of [Automatic installation of Recommended extensions is failing on Java Samples.](https://issues.redhat.com/browse/CRW-4790) test failed by common timeout common 20min: `MOCHA_DEFAULT_TIMEOUT: process.env.MOCHA_DEFAULT_TIMEOUT || 1200000`
Added specific timeout test timeout 
`TS_FIND_EXTENSION_TEST_TIMEOUT * n`, where `n` - number of extensions.
15s per extension by default.
- added possibility to run local allure reporter
- updated performTrustAuthorDialog() method to avoid unnecessary imports 
- small fixes

### Screenshot/screencast of this PR

NA

### What issues does this PR fix or reference?

#22502 
https://issues.redhat.com/browse/CRW-4790

### How to test this PR?

`docker run --shm-size=4096m -p 5920:5920 -e TS_SELENIUM_LOAD_PAGE_TIMEOUT=420000 -e TS_SELENIUM_BASE_URL=https://devspaces.apps.ocp413.crw-qe.com/ -e DELETE_WORKSPACE_ON_FAILED_TEST=true -e TS_SELENIUM_START_WORKSPACE_TIMEOUT=120000 -e NODE_TLS_REJECT_UNAUTHORIZED=0 -e VIDEO_RECORDING=true -e E2E_OCP_CLUSTER_API_URL=https://api.ocp413.crw-qe.com:6443/ -e TS_SELENIUM_LOG_LEVEL=TRACE -e TS_WAIT_LOADER_PRESENCE_TIMEOUT=120000 -e TS_COMMON_DASHBOARD_WAIT_TIMEOUT=30000 -e OCP_INFRA=PSI -v /tmp:/usr/local/bin:Z -v /home/hudson/.kube:/home/seluser/.kube:Z -v /mnt/hudson_workspace/workspace/Testing/e2e/basic/typescript-tests/test-repo/tests/e2e:/tmp/e2e:Z -v /mnt/hudson_workspace/workspace/Testing/e2e/basic/typescript-tests/test-repo/tests/e2e/video:/tmp/ffmpeg_report:Z -e USERSTORY=RecommendedExtensions -e TS_SELENIUM_VALUE_OPENSHIFT_OAUTH=true -e TS_OCP_LOGIN_PAGE_PROVIDER_TITLE=htpasswd -e TS_SELENIUM_OCP_USERNAME=admin -e 'TS_SAMPLE_LIST=Java 11 with Quarkus' -e TS_COMMON_DASHBOARD_WAIT_TIMEOUT=120_000 quay.io/eclipse/che-e2e:next`

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix or new feature ](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix-or-new-feature)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Review by musienko-maxim — APPROVED** [review]

**Comment by nallikaea** [comment]
https://main-jenkins-csb-crwqe.apps.ocp-c1.prod.psi.redhat.com/job/Testing/job/e2e/job/basic/job/typescript-tests/11960/
run time before fail was reduced as expected

File: tests/e2e/constants/PLUGIN_TEST_CONSTANTS.ts [hunk 1]
```diff
@@ -10,6 +10,9 @@
 export const PLUGIN_TEST_CONSTANTS: { TS_SAMPLE_LIST: string } = {
 	/**
 	 * dashboard samples to check in RecommendedExtensions.spec.ts
+	 * full list Node.js MongoDB,Node.js Express,Java 11 with Lombok,Java 11 with Quarkus,Python,.NET,C/C++,Go,PHP,Ansible
```
Comment (dmytro-ndp) [inline tests/e2e/constants/PLUGIN_TEST_CONSTANTS.ts]: this sentence looks redundant because is duplicating TS_SAMPLE_LIST constant value

**Review by dmytro-ndp — APPROVED** [review]
looks good to merge taking into account shared test results: https://github.com/eclipse/che/pull/22514#issuecomment-1727311371

**Comment by dmytro-ndp** [comment]
@nallikaea: new test run showed the expected result: https://main-jenkins-csb-crwqe.apps.ocp-c1.prod.psi.redhat.com/job/Testing/job/e2e/job/basic/job/typescript-tests/12045/console

**Comment by dmytro-ndp** [comment]
EmptyWorkspace PR check has passed as well https://github.com/eclipse/che/actions/runs/6258850326/job/16993691751?pr=22514

**Pull request merged**